### PR TITLE
Log trace changes to aling with unified format for IoTP

### PR DESCRIPTION
**Pull request by fgalan**
Implements #2353

### CODE REVIEW

File: src/lib/rest/rest.cpp [hunk 1]
```diff
@@ -1141,16 +1142,20 @@ static int connectionTreat
     //
     lmTransactionStart("from", ip, port, url);  // Incoming REST request starts
 
-
-    /* X-Forwared-For (used by a potential proxy on top of Orion) overrides ip */
-    if (ciP->httpHeaders.xforwardedFor == "")
+    /* X-Real-IP and X-Forwared-For (used by a potential proxy on top of Orion) overrides ip.
+       X-Rreal-IP takes preference over X-Forwared-For, if both appear */
```
Comment (kzangeli) [inline src/lib/rest/rest.cpp]: typo: Rreal

Comment (fgalan) [inline src/lib/rest/rest.cpp]: Fixed in dca40219931c1b9caebdcf7527722e15b6142642

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**